### DISCUSSION TITLE
Fulfillment Center (FC) transfers

### DISCUSSION THREAD

**Discussion by Holdinar** [amzn/selling-partner-api-models#3608] [General]
I need help with Fulfillment Center (FC) transfers. Our objective is to identify the specific stock that is currently being transferred between FCs, and to see the source warehouse and destination warehouse. Despite downloading numerous reports and making several requests, I am still unable to access the required information. If anyone could provide help or guidance, it would be greatly appreciated.

**Comment by Pallettb** [comment]
Did you find the answer? I tried everything, too. It should be "pendingTransshipmentQuantity", but it comes back with 0 for me.

**Reply by Holdinar** [reply]
No, I even contacted Amazon, and they told me I need to request information through chat to figure out which warehouses are the destination and source. They told me I need to contact them for each log which is so stupid and frustrating. The only thing I could find is Amazon FC fulfillment or inventory ledger that only shows warehouse ID transferring 5 items to some unknown warehouse. Then, a few days later, you'd get a new log of items received at some warehouse. However, with a large log of items going left and right, it's challenging to determine which is which. Still a big issue for automatic financing, amazon is stupid.
